src/Voron/StorageEnvironmentOptions.cs

```diff
@@ -502,39 +495,44 @@ private string[] GetRecyclableJournalFiles()
                 }
                 catch (Exception)
                 {
-                    return new string[0];
+                    return [];
                 }
             }
 
-            public VoronPathSetting FilePath { get; private set; }
+            public VoronPathSetting FilePath { get; }
+
+            public override (Pager Pager, Pager.State State) InitializeDataPager()
+            {
+                var flags = Pal.OpenFileFlags.None;
+                if(Encryption.IsEnabled)
+                    flags |= Pal.OpenFileFlags.Encrypted;
```

Comment:
It instructs the `PAL` to lock the data in memory

Reply:
Sorry, I was wrong. The actual usage is to mark it for the pager, so it will know to use the crypto functions:

https://github.com/ayende/ravendb/blob/RavenDB-22457/src/Voron/Impl/Paging/Pager.cs#L75

Reply:
OK. Makes sense now